Arm/source/main.cpp

```diff
@@ -0,0 +1,47 @@
+#include "utility/log.hpp"
+#include "RoverArmSystem.hpp"
+#include "L1_Peripheral/lpc40xx/i2c.hpp"
+#include "L1_Peripheral/lpc40xx/can.hpp"
+#include "L2_HAL/actuators/servo/rmd_x.hpp"
+#include "L2_HAL/sensors/movement/accelerometer/mpu6050.hpp"
+
+int main()
+{
+  // Create a can bus for the RMD_x7 motors to use and all of the RMD_x7 motor
+  // objects for the arm.
+  sjsu::lpc40xx::Can can(sjsu::lpc40xx::Can::Channel::kCan2);
+  sjsu::StaticAllocator<1024> memory_resource;
+  sjsu::CanNetwork can_network(can, &memory_resource);
+  sjsu::RmdX rmd_rotunda(can_network, 0x148);
+  sjsu::RmdX rmd_shoulder(can_network, 0x149);
+  sjsu::RmdX rmd_elbow(can_network, 0x14A);
+  sjsu::RmdX rmd_left_wrist(can_network, 0x14B);
+  sjsu::RmdX rmd_right_wrist(can_network, 0x14C);
+
+  // Create an I2C object for the IMU's as well as all of the MPU objects for
+  // the arm.
+  sjsu::lpc40xx::I2c i2c(sjsu::lpc40xx::I2c::Bus::kI2c2);
+  sjsu::Mpu6050 mpu_rotunda(i2c, 0x68);
+  sjsu::Mpu6050 mpu_shoulder(i2c, 0x69);
+  sjsu::Mpu6050 mpu_elbow(i2c, 0x6A);
+  sjsu::Mpu6050 mpu_wrist(i2c, 0x6B);
```

Comment:
Place a comment here stating that this may not work depending on if we use the address translator or if we use the chip select method. Will go over this in the arm meeting.